src/variety/haisu.js

```diff
@@ -438,7 +438,7 @@
 			"The line goes through S/G"
 		],
 		haisuError: [
-			"(please translate) A number is not passed on the right visit",
+			"数字を正しい回数目で通過していません。",
```

Comment:
About this propose translation, the proposer says that it is not very good, and I think so.
but, I don't have any other good idea:(

In my opinion, **it's better to discard this translation and waiting for another one.**

Reply:
I think the English message is also not very good, so I wouldn't worry too much about whether it's a good translation, only about whether it explains the rule...

Reply:
I see. but, this proposed translation is not also good explanation of the rule....
I understand the meaning of this rule, but cannot explain it in simple Japanese X(

After merging other trasnlations, I'd like to think about the explanation in Japanse again.